standard/types.md

```diff
@@ -601,8 +601,20 @@ A type parameter is an identifier designating a value type or reference type tha
 type_parameter
     : identifier
     ;
+
+nullable_type_parameter
+    : non_nullable_non_value_type_parameter '?'
+    ;
+
+non_nullable_non_value_type_parameter
+    : type_parameter
+    ;
```

Comment:
This grammar is wrong/pointless, as @KalleOlaviNiemitalo has commented the added rules are not used anywhere.

Unfortunately I’m not sure what the intended semantics are here.

Guessing: is the addition of the nullable type attribute meant to require the type argument to be a nullable reference and/or value type, i.e. it’s an additional kind of *primary_constraint* or *secondary_constraint*?

Reply:
That was my mistake. I added the additional rules in the latest commit.

Note that the grammar additions for `non_nullable_reference_type` and `nullable_reference_type` are in #1089